LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -96,7 +96,11 @@ public final class GlucoseStore: HealthKitSampleStore {
     }
     private let lockedLatestGlucose = Locked<GlucoseSampleValue?>(nil)
 
-    private let cacheStore: PersistenceController
+    public let cacheStore: PersistenceController
```

Comment:
Ah, thanks, this can be reverted. `cacheStore` is passed in via the public constructor, so it's not really isolated to the internal implementation, but I'll revert this, as it's not needed; I think it was part of my attempt to test at one point.